CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Refactored `NewComponentButton` (#450)
 - Modal styling (#450)
 - Projects tab layout (#447)
+- Position run button in separate bar (#471)
```

Comment:
⛏️ needs to go in the unreleased section 👍

Reply:
ooops! have fixed